data/db/slr-nikon.xml

```diff
@@ -3060,22 +3060,38 @@
     <lens>
         <maker>Nikon</maker>
         <model>Nikon AF Zoom-Nikkor 28-200mm f/3.5-5.6G IF-ED</model>
-        <model lang="en">Nikkor AF 28-200mm f/3.5-5.6G IF-ED</model>
         <mount>Nikon F AF</mount>
         <cropfactor>1</cropfactor>
         <calibration>
             <!-- Taken with Nikon D600 -->
-            <distortion model="ptlens" focal="28" a="0.08017" b="-0.26356" c="0.26077"/>
-            <distortion model="ptlens" focal="42" a="0.01706" b="-0.03987" c="0.03912"/>
-            <distortion model="ptlens" focal="66" a="0.00718" b="-0.00425" c="0.0168"/>
-            <distortion model="ptlens" focal="200" a="0" b="0.01864" c="0"/>
-            <tca model="poly3" focal="28" vr="1.0002275" vb="1.0000944"/>
-            <tca model="poly3" focal="42" vr="1.0000854" vb="1.0000926"/>
-            <tca model="poly3" focal="50" vr="1.0000529" vb="1.0001110"/>
-            <tca model="poly3" focal="66" vr="0.9999807" vb="1.0001296"/>
-            <tca model="poly3" focal="95" vr="0.9999182" vb="1.0000201"/>
-            <tca model="poly3" focal="135" vr="0.9998602" vb="0.9999489"/>
-            <tca model="poly3" focal="200" vr="0.9999000" vb="0.9999004"/>
+            <distortion model="ptlens" focal="28" a="0.03547" b="-0.10778" c="0.08687" />
+            <distortion model="ptlens" focal="38" a="0.02089" b="-0.04522" c="0.02617" />
+            <distortion model="ptlens" focal="48" a="0.2973" b="0.67995" c="-0.12197" />
+            <distortion model="ptlens" focal="62" a="0.0182" b="-0.03995" c="0.04876" />
+            <distortion model="ptlens" focal="85" a="0.01073" b="-0.00974" c="0.0161" />
+            <distortion model="ptlens" focal="125" a="0.00065" b="0.03125" c="-0.03952" />
+            <distortion model="ptlens" focal="200" a="0.01851" b="-0.05192" c="0.08089" />
+            <tca model="poly3" focal="28" vr="1.0002152" vb="1.0000092" />
+            <tca model="poly3" focal="38" vr="1.0001291" vb="1.0000999" />
+            <tca model="poly3" focal="48" vr="1.0000649" vb="1.0000876" />
+            <tca model="poly3" focal="62" vr="0.9999990" vb="1.0001145" />
+            <tca model="poly3" focal="85" vr="0.9999428" vb="1.0000457" />
+            <tca model="poly3" focal="125" vr="0.9999015" vb="0.9999823" />
+            <tca model="poly3" focal="200" vr="0.9999001" vb="0.9998229" />
+            <vignetting model="pa" focal="28" aperture="4.5" distance="10" k1="0.6082" k2="-2.2120" k3="1.3423" />
```

Comment:
Please remove the vignetting data.  Vignetting images always must be made through a diffuser (e.g. paper or milk glass).  Also, remove the “vignetting” directory.